src/components/annotations/attributes.js

```diff
@@ -41,19 +42,23 @@ module.exports = {
         valType: 'angle',
         dflt: 0,
         role: 'style',
+        editType: 'docalcAutorange',
         description: [
             'Sets the angle at which the `text` is drawn',
             'with respect to the horizontal.'
         ].join(' ')
     },
-    font: extendFlat({}, fontAttrs, {
+    font: fontAttrs({
+        editType: 'docalcAutorange',
+        colorEditType: 'doarraydraw',
```

Comment:
Could you point me toward what `doarraydraw` does? It's not entirely clear from the name, and if it appears somewhere else in the source, it's not jumping out at me.

Reply:
Never mind. Found it in the suppressed diff.

Reply:
Wait, where's is `doarraydraw` being used? I can't find it (even in the suppressed diffs :upside_down_face: )

Reply:
I think at one point I had `none` instead of `doarraydraw` but that looked weird. Then at another point I had a block just for debugging that checked that everything labeled `doarraydraw` was captured by an `arrayContainers` block and threw an error otherwise, but I removed that once I was satisfied that it was never getting hit... happy to change it if you think it would make more sense another way.

Reply:
If I understand correctly, `doarraydraw` is just a placeholder for edits in array containers that are handled by `applyContainerArrayChanges` in [`manage_array.js`](https://github.com/plotly/plotly.js/blob/11bb997e78128b9f47db2ed392982f82a026e589/src/plot_api/manage_arrays.js#L73). 

In this case, I prefer setting `editType: 'doarraydraw'` over `editType: 'none'`. Just make sure that this behavior is documented in the edit type `description` field (cc https://github.com/plotly/plotly.js/pull/1999#discussion_r138708703).